tensorboard/__init__.py

```diff
@@ -22,7 +22,7 @@
 from tensorboard import lazy
 
 pkg = lambda i: i  # helps google sync process
-mod = lambda i: lazy.LazyLoader(i[i.rindex('.') + 1:], globals(), i)
+mod = lambda i: lazy.LazyLoader(i[i.rindex('.') + 1:], globals(), i)  # noqa
```

Comment:
Can you specify in the suppress tag the specify error to suppress e.g.s `# noqa: E731` if that's what it was?

Reply:
What is the max-line-length that you want to set?